src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,61 @@
+// import { type } from 'os';
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movies: Movie;
```

Comment:
i guess it's a single movie here
```suggestion
  movie: Movie;
```